content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -223,18 +246,48 @@ plus a few additional polyfills:
   - `browserify-zlib`
   - `path-browserify`
   - `react-native-url-polyfill`
+  - `react-native-quick-crypto`
+
+```bash
+  # Using npm
+  npm install @metaplex-foundation/js@0.19.4 \
```

Comment:
`@metaplex-foundation/js` is deprecated and won't work anymore. See https://earn.superteam.fun/listings/bounty/update-solana-developer-lessons/. If you need demo code for the current Metaplex try https://github.com/solana-developers/professional-education/tree/main/labs/metaplex-umi